src/include/batch_request.h

```diff
@@ -410,7 +403,7 @@ extern void  req_confirmresv(struct batch_request *req);
 extern void  req_connect(struct batch_request *req);
 extern void  req_defschedreply(struct batch_request *req);
 extern void  req_locatejob(struct batch_request *req);
-extern void  req_manager(struct batch_request *req);
+extern void  req_manager(struct batch_request *req, conn_t *conn);
```

Comment:
Why is req_manager the only IFL api handler that needs a conn_t *?

Reply:
@subhasisb Thanks for catching this, reverted signature...